Client/Info.plist

```diff
@@ -148,8 +148,13 @@
 	</array>
 	<key>UIFileSharingEnabled</key>
 	<true/>
-	<key>UILaunchStoryboardName</key>
-	<string>LaunchScreen</string>
+	<key>UILaunchScreen</key>
+	<dict>
+		<key>UIColorName</key>
```

Comment:
Nice

Reply:
yeah I haven't know this API before today: here's the article explaining it: https://www.avanderlee.com/xcode/launch-screen/

Reply:
Interesting, is it compatible with older iOS versions?

Reply:
no 😭 - just checked.. seems to only work for iOS14 and up 😢 will try to find a workaround